test/config/group3/case1.yaml

```diff
@@ -0,0 +1,149 @@
+- method: SetNodeToBad
+  parameters:
+    nodeName: 0.0.0.5
+- method: SetNodeToBad
+  parameters:
+    nodeName: 0.0.0.6
+- method: SetNodeToBad
+  parameters:
+    nodeName: 0.0.0.7
+- method: SchedulePod
+  parameters:
+    podName: group1_pod1
+    phase: "Preempting"
+    podGroupSchedulingRequest:
+      version: v2
+      virtualCluster: VC1
+      priority: 1
+      pinnedCellId: ""
+      cellType: V100
+      cellNumber: 1
+      gangReleaseEnable: false
+      lazyPreemptionEnable: false
+      podRootGroup:
+        name: group1
+        withinOneCell: ""
```

Comment:
within V100-RACK? otherwise it will be scheduled to 0.0.0.8 to reduce fragmentation